docs/ref/contrib/postgres/functions.txt

```diff
@@ -0,0 +1,26 @@
+PostgreSQL specific database functions
+======================================
+
+All of these fields and widgets are available from the
+``django.contrib.postgres.functions`` module.
+
+.. currentmodule:: django.contrib.postgres.functions
+
+TransactionNow
+--------------
+
+.. class:: TransactionNow()
+
+.. versionadded:: 1.9
+
+Returns the date and time on the database server that the current transaction
+started. If you are not in a transaction it will return the date and time of
+the current statement. This is a complement to
+:class:`django.db.models.functions.Now`, which always returns the date and time
+of the current statement.
```

Comment:
Good point - it's the outmost, i.e. between `BEGIN` and `COMMIT` - I guess the others are technically savepionts:

```
djangotests=# begin;
BEGIN

djangotests=# select current_timestamp, statement_timestamp();
              now              |      statement_timestamp      
-------------------------------+-------------------------------
 2015-06-03 10:53:22.323409+01 | 2015-06-03 10:53:32.522866+01
(1 row)

djangotests=# savepoint alpha;
SAVEPOINT
djangotests=# select current_timestamp, statement_timestamp();
              now              |      statement_timestamp      
-------------------------------+-------------------------------
 2015-06-03 10:53:22.323409+01 | 2015-06-03 10:53:40.771233+01
(1 row)
```

Looks like there isn't a savepoint timestamp function [in the manual](http://www.postgresql.org/docs/8.2/static/functions-datetime.html), you'd have to save it yourself in a variable.